doc/releases/release-notes-4.0.rst

```diff
@@ -220,6 +223,7 @@ Boards & SoC Support
    * :zephyr:board:`WeAct Studio STM32F405 Core Board V1.0 <weact_stm32f405_core>` (``weact_stm32f405_core``)
    * :zephyr:board:`WeAct Studio USB2CANFDV1 <usb2canfdv1>` (``usb2canfdv1``)
    * :zephyr:board:`Witte Technology Linum Board <linum>` (``linum``)
+   * :zephyr:board:`Seeed XIAO ESP32-S3 Sense Variant <xiao_esp32s3>`: ``xiao_esp32s3``.
```

Comment:
@kartben This is a variant. Let me know if this is the right place to do so.

Reply:
It's fine to add, and actually a good idea. Please just keep alphabetical order intact though, and consider indicating the full target name that includes the variant (it's a bit of a mouthful though:))

Reply:
``xiao_esp32s3/esp32s3/procpu/sense``
But maybe it's too long, like I said :)